pkg/controllers/jobtemplate/jobtemplate_controller.go

```diff
@@ -81,14 +80,14 @@ func (jt *jobtemplatecontroller) Initialize(opt *framework.ControllerOption) err
 	jt.kubeClient = opt.KubeClient
 	jt.vcClient = opt.VolcanoClient
 
-	jt.jobTemplateInformer = informerfactory.NewSharedInformerFactory(jt.vcClient, 0).Flow().V1alpha1().JobTemplates()
+	jt.jobTemplateInformer = opt.VCSharedInformerFactory.Flow().V1alpha1().JobTemplates()
```

Comment:
Welcome! Please change to `factory := opt.VCSharedInformerFactory` to keep consistent with other controllers.

Reply:
done